extensions/jitsi/common/jitsi.mjs

```diff
@@ -412,6 +429,8 @@ class JitsiHandler {
       }
     }
 
+    this.broadcastParticipantId(null); // broadcast that the player has quit.
```

Comment:
Maybe await here or at the end of the function to reduce the chance of race condition?